### ISSUE TITLE
implement Labels

### ISSUE THREAD

**Issue by rolph-recto** [apl-cornell/viaduct#3]
implement a `FreeDistributedLattice<G>` class parameterized over a generator set G, and then the `Label` class would look like:

```Java
class Label implements Lattice<Label> {
  FreeDistributedLattice<Principal> confidentiality;
  FreeDistributedLattice<Principal> integrity;
}
```

**Comment by cacay** [comment]
The official term is Distribu**tive** lattice, not distributed.

**Comment by rolph-recto** [comment]
resolved as of 9aa9372aa1dd22b59f426d1acab796193575f006
